spec/widgets/category/search-title-view.spec.js

```diff
@@ -35,7 +36,6 @@ describe('widgets/category/search-title-view', function () {
       expect($el.find('.js-title').length).toBe(1);
       expect($el.find('.CDB-Widget-options').length).toBe(1);
       expect($el.find('.js-titleText').length).toBe(1);
-      expect(this.view.$('.js-autoStyle').length).toBe(1);
```

Comment:
~~Suggestion: Maybe it's better to stub the `_isAutoStyleButtonVisible` (like below) rather than removing this check?~~

Edit: Disregard, this is tested in depth below.